Makefile

```diff
@@ -24,27 +25,40 @@ clean-pyc:
 	find . -name '__pycache__' -exec rm -rf {} +
 
 lint:
-	tox -e lint
-
-lint-roll:
-	isort eth_bloom tests
-	black eth_bloom tests setup.py
-	$(MAKE) lint
+	@pre-commit run --all-files --show-diff-on-failure || ( \
+		echo "\n\n\n * pre-commit should have fixed the errors above. Running again to make sure everything is good..." \
+		&& pre-commit run --all-files --show-diff-on-failure \
+	)
 
 test:
 	pytest tests
 
-test-all:
-	tox
+build-docs: docs
```

Comment:
This doesn't seem useful if it just calls `docs` 🤔... does this facilitate something I'm not seeing when the template is applied?

Reply:
It's there so that the `notes` and `release` commands don't need to change from the template.

Reply:
Can be changed if you prefer, I'm 50/50.

Reply:
I figured it was something like that. It seems fine tbh, as long as we do it similarly on all CHANGELOG repos so we don't start deviating too much from a known pattern. I believe `py-geth` is set up this way. Can we mimic that repo here?

Reply:
> Changed it, won't be an issue in future template merges, and looks cleaner.

Oops I missed this when I replied. That was quick. That makes sense. We should change it in *py-geth* as well when we get the chance.